packages/suite/src/utils/wallet/accountUtils.ts

```diff
@@ -486,5 +486,14 @@ export const accountSearchFn = (
           account.addresses.change.find(matchAddressFn)
         : false;
 
-    return symbolMatch || networkNameMatch || accountTypeMatch || descriptorMatch || addressMatch;
+    const metadataMatch = account.metadata.accountLabel?.includes(searchString);
```

Comment:
startsWith would be more convenient for sure!

Reply:
good catch!